extension-manifest-v3/src/utils/api.js

```diff
@@ -66,6 +100,20 @@ export async function session() {
 
   let accessToken = await getCookie('access_token');
```

Comment:
shouldn't you extract the cookie value here? as it was in https://github.com/ghostery/ghostery-extension/pull/1259/files#diff-c917c022c9d3fb48f76186de6bbbed139cc1e135fcbfd191fa76dd7408483371L48